content/en/about/contributing/shortcodes/_index.md

```diff
@@ -144,102 +142,127 @@ in multiple programming languages, you can use a tabbed pane for code snippets:
 
 Will render as:
 
+<!-- prettier-ignore -->
 {{< tabpane >}}
-{{< tab header="C++" lang="C++">}}
-#include <iostream>
-int main()
-{
-  std::cout << "Hello World!" << std::endl;
-}
-{{< /tab >}}
-{{< tab header="Java" lang="Java">}}
-class HelloWorld {
-  static public void main( String args[] ) {
-    System.out.println( "Hello World!" );
-  }
-}
-{{< /tab >}}
-{{< tab header="Kotlin" lang="Kotlin">}}
-fun main(args : Array<String>) {
-    println("Hello, world!")
-}
-{{< /tab >}}
-{{< tab header="Go" lang="Go">}}
-package main
-import "fmt"
-func main() {
-  fmt.Printf("Hello World!\n")
-}
-{{< /tab >}}
+  {{< tab header="C++" lang="C++" >}}
+    #include <iostream>
+    int main()
+    {
+        std::cout << "Hello World!" << std::endl;
+    }
+  {{< /tab >}}
+  {{< tab header="Java" lang="Java" >}}
+    class HelloWorld {
+        static public void main( String args[] ) {
+            System.out.println( "Hello World!" );
+        }
+    }
+  {{< /tab >}}
+  {{< tab header="Kotlin" lang="Kotlin" >}}
+    fun main(args : Array<String>) {
+        println("Hello, world!")
+    }
+  {{< /tab >}}
+  {{< tab header="Go" lang="Go" >}}
+    import "fmt"
```

Comment:
these spaces are rendered also, so it's not a good thing for python

Reply:
Good point, the indentation is causing troubles (as it was for the tabpanex), just removed it.